core/bin/zksync_api/src/fee_ticker/validator/mod.rs

```diff
@@ -188,6 +190,20 @@ mod tests {
         }
     }
 
+    #[tokio::test]
+    #[ignore]
+    // We can use this test only online, run it manually if you need to test connection to coingecko
```

Comment:
what do you mean with online?

Reply:
1. it is the original comment (and test), where I only changed uniswap to coingecko
2. to me this means that it shouldn't be run in the test suite, instead it should only be used to test the API connection, manually